### DISCUSSION TITLE
Upcoming OpenDDS Version 3.30.0

### DISCUSSION THREAD

**Discussion by iguessthislldo** [OpenDDS/OpenDDS#4817] [Announcements]
The OpenDDS Foundation plans to release the master branch as OpenDDS 3.30.0 in the near future. The current release notes can be found [here](https://opendds.readthedocs.io/en/master/news.html#v3-30-0). Any feedback would be greatly appreciated.

**Comment by jwillemsen** [comment]
[AXCIOMA](https://www.axcioma.org) with OpenDDS master seems to compile and run as it should, see our [scoreboard](https://download.remedy.nl/buildlogs/fc_axcioma_scoreboard/).
